src/test/integration/integrationtests.py

```diff
@@ -162,6 +163,37 @@ def test_parking(self):
 
         self.assertEqual(parking_zones, result_parking_zones)
 
+    # Test results returned with isOpen=true are actually open given what their openHours say
+    def test_isopen(self):
+        def test_resource(resource_url):
+            all_open_resources = query_request(
+                resource_url,
+                access_token,
+                'get',
+                {'page[size]': max_page_size, 'isOpen': 'true'}
+            ).json()
+
+            now = datetime.utcnow().replace(microsecond=0).isoformat()
+            weekday = str(datetime.today().weekday() + 1)
+
+            for open_resource in all_open_resources['data']:
+                # Test that only open resources are returned when they should be and each open resource has a related open hours
+                open_hours = open_resource['attributes']['openHours']
+                self.assertIsNotNone(open_hours)
+
+                # If there are not only one open hours, current time should be in one of these open hours intervals
+                if len(open_hours[weekday]) > 1:
+                    is_open = False
+                    for open_hour in open_hours[weekday]:
+                        if open_hour['start'][:-1] < now < open_hour['end'][:-1]:
```

Comment:
is the `[:-1]` to strip the trailing Z? can we do that explicitly instead? `open_hour['start'].rstrip('Z')`. or just add a trailing Z to `now`.

Reply:
Agree. Will fix.